tools/src/test/java/org/apache/kafka/tools/consumer/group/DeleteOffsetsConsumerGroupCommandIntegrationTest.java

```diff
@@ -88,7 +113,7 @@ public void testDeleteOffsetsNonExistingGroup() {
         }
     }
 
-    @ClusterTest
+    @ClusterTemplate("generator")
     public void testDeleteOffsetsOfStableConsumerGroupWithTopicPartition() {
         for (Map<String, Object> consumerConfig: consumerConfigs) {
```

Comment:
`consumerConfigs` need to be rewritten by `ClusterInstance#supportedGroupProtocols`

see https://github.com/apache/kafka/pull/15766/files#r1590222192

Reply:
Updated it. Thanks.